src/app.js

```diff
@@ -13,7 +13,23 @@ export default class App {
         this.router.addRoute('/weather', new Weather());
         this.router.addRoute('/football', new Sport());
         this.router.addRoute('/news', new News());
-        this.router.addRoute('/health', new Health());
         this.router.start();
+        this._displayMobileMenu();
```

Comment:
please check if we are using mobile device or not. If yes then render burger menu